src/test/ui/weird-exprs.rs

```diff
@@ -159,6 +160,11 @@ fn match_nested_if() {
     assert!(val);
 }
 
+fn empty_tuple_assignment() {
```

Comment:
I think this should match the naming convention of the file it's in ;)

```suggestion
fn monkey_barrel() {
```